scripts/build.py

```diff
@@ -240,6 +244,9 @@ def cmake_generate_options(args):
             '-DBUILD_LAUNCHER_AND_INTERCEPTOR={}'.format(
                 'OFF' if not args.build_launcher else 'ON'))
         generate_options.extend('-D' + arg for arg in args.cmake_extra)
+        if args.generator:
+            generate_options.append('-G{}'.format(args.generator))
+            generate_options.append("-DCMAKE_CXX_FLAGS=-fdiagnostics-color=always")
```

Comment:
I'm not sure this should be the default - is this line a leftover from changes local to your machine?